### PR TITLE
Prepare and add popup type support

### PR DESCRIPTION
**Pull request by msisov**
This PR adds a popup type support.

### CODE REVIEW

File: ui/ozone/platform/wayland/wayland_window.cc [hunk 1]
```diff
@@ -130,6 +119,33 @@ bool WaylandWindow::Initialize() {
   return true;
 }
 
+void WaylandWindow::CreateXdgPopup() {
+  if (!parent_window_)
+    GetParentWindow();
```
Comment (tonikitoo) [inline ui/ozone/platform/wayland/wayland_window.cc]: missing 'return'?

Comment (msisov) [reply ui/ozone/platform/wayland/wayland_window.cc]: No, return is not needed.

**Review by tonikitoo — COMMENTED** [review]
nit: missing "!" after fixup.

File: ui/ozone/platform/wayland/wayland_window.cc (hunk 1 above)
Comment (tonikitoo) [reply ui/ozone/platform/wayland/wayland_window.cc]: it seems then that the GetParentWindow "populates" parent_window_, so name isn't ideal.

**Review by tonikitoo — APPROVED** [review]

File: ui/ozone/platform/wayland/wayland_window.cc (hunk 1 above)
Comment (msisov) [reply ui/ozone/platform/wayland/wayland_window.cc]: ok!

**Review by tonikitoo — APPROVED** [review]
lgtm

**Pull request merged**